src/shillelagh/backends/apsw/vt.py

```diff
@@ -317,8 +352,14 @@ def BestIndex(  # pylint: disable=too-many-locals
                     indexes.append((column_index, sqlite_index_constraint))
                     break
             else:
-                # no indexes supported in this column
-                constraints_used.append(None)
+                if (operator == Operator.LIMIT and self.adapter.supports_limit) or (
```

Comment:
I think you should use `is` here rather than `==`

Reply:
Ah, good catch, let me fix it.